### ISSUE TITLE
Death to Tenants! Long live Organizations!

### ISSUE THREAD

**Issue by mheggeseth** [HTBox/allReady#423]
The initial rev of allReady referred to humanitarian organizations using the site as "tenants" which is a wonky, non-obvious term. It makes more sense to refer to them as "organizations" or "orgs" for short where appropriate.
* At the very least, the word "tenant" should not appear anywhere in the UI. Currently it shows up on `/Admin/Login`.
* As a bonus, change "tenant" code identifiers to "organization". This is easy for local variables, method names, folders, URLs, and other code identifiers that don't map to a database table or column. Changing names of database entities ~~should probably be a~~ is separate issue (#426) and warrants more discussion.

cc @MisterJames

**Comment by stevejgordon** [comment]
Out of curiosity I searched the code for tenant and it's used 924 times across 127 files. This includes js files also. That would likely make one very large commit. I can start with the UI items and look at the other bits at some stage when time permits. Do we want to go all the way and rename the js level as well? It feels like the time to make this change if it needs to happen and would avoid confusion to other devs later on.

We should though agree on capitalisation rules. Currently it reads as "Register as a new Tenant co-ordinator?". Should Organization also be capitalised in such use cases?

**Comment by stevejgordon** [comment]
Forgot to mention that also there is the TenantAdmin claim type. If we were to rename this then any user with that claim in the DB currently would need to be updated.

**Comment by mheggeseth** [comment]
I think do it all at once and go all the way. You're going to cause merge conflicts with people however you split it up, so you might as well do it all at once. However, I think you should do all the DB-related stuff first since that will require more care and coordination (everyone should drop and re-create their AllReady database) and we don't want to miss our window on that (I don't know what the window is, but @tonysurma made it sound on the order of weeks).

Re: punctuation. I don't think "organization" should be considered a proper noun so it should not have special casing rules. Remember, though, (especially in code) that we will use the U.S. spelling (with a z).

**Comment by stevejgordon** [comment]
I plan to have a crack at the final eradiation of the tenant wording through the classes and properties once my PR #456 for the DB changes is all merged in.

**Comment by stevejgordon** [comment]
I have started on this one locally

**Comment by stevejgordon** [comment]
I have this all ready to go. I'll wait on PR #459 since it will introduce some slight changes I'll need to make to this before submitting the PR.
